twitter/View Controllers/ComposeViewController.m

```diff
@@ -0,0 +1,53 @@
+//
+//  ComposeViewController.m
+//  twitter
+//
+//  Created by Nicholas Wayoe on 6/30/20.
+//  Copyright © 2020 Emerson Malca. All rights reserved.
+//
+
+#import "ComposeViewController.h"
+#import "APIManager.h"
+
+@interface ComposeViewController ()
+@property (weak, nonatomic) IBOutlet UITextView *tweetView;
+
+@end
+
+@implementation ComposeViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+- (IBAction)onTapTweetButton:(id)sender
+{
+    NSString *tweetContent = self.tweetView.text;
+    [[APIManager shared] postStatusWithText:tweetContent completion:^(tweet *newTweet, NSError *error) {
```

Comment:
nit: `APIManager.shared`

Reply:
Also love that you called it newTweet. Good call.

Reply:
will make the changed. But why do you prefer the dot syntax of the [ ] one

Reply:
Generally better to access properties with dot notation. Mostly just that square brackets are hard to read so the fewer the better imo.